src/js/services/forms.js

```diff
@@ -210,19 +240,32 @@ export default App.extend({
 
     return Promise.all([
       Radio.request('entities', 'fetch:forms:definition', this.form.id),
-      Radio.request('entities', 'fetch:formResponses:submission', responseId),
+      Radio.request('entities', 'fetch:formResponses:model', responseId),
     ]).then(([definition, response]) => {
       channel.request('send', 'fetch:form:response', {
         definition,
-        formSubmission: get(response, 'data', {}),
+        formSubmission: response.getResponse(),
         contextScripts: this.form.getContextScripts(),
       });
     });
   },
+  /* istanbul ignore next: Testing the debounce is buggy with cy.clock */
```

Comment:
@nmajor25 I spent forever trying to figure out a reasonable way to test this with the 30 second debounce.  `cy.clock` doesn't work reliably because it doesn't always pass the `defer` that calls `Backbone.History.start` and everything fails.. I have some ideas to fix this, but it has to be in a future release.

Reply:
I just realized we're missing some other coverage of functions that only this calls.